src/App.js

```diff
@@ -0,0 +1,31 @@
+import { Routes, Route, Link } from 'react-router-dom';
+import BookList from './components/Book-list';
+import StatuseButton from './components/categories';
+import InputForm from './components/Input-form';
+import './components/styles/Navbar.css';
+
+function App() {
+  return (
+    <div>
+      <h1>Bookstore CMS</h1>
+      <ul className="form">
+        <li><Link to="/">Books</Link></li>
+        <li><Link to="/StatusButton">Categories</Link></li>
+      </ul>
+      <Routes>
+        <Route
+          path="/"
+          element={(
+            <div>
+              <BookList />
+              <InputForm />
+            </div>
```

Comment:
As part of the requirements the app should have 2 main components Books & Categories, 
**_Books:_**
- [ ] kindly Create a Books component that contains both  `<BookList /> & <InputForm />` and update your route to be `<Route path="/Status" element={<Books/>} />`


**_Categories_**

- [ ] the categories path should be ` path="/categories"` instead of `path="/StatusButton"` , kindly fix it 🙏 

- [ ] To make your code more readable kindly rename the `<StatuseButton />` component to `<Categories/>`